src/app/containers/manuscript/index.tsx

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
-import { Backdrop, CircularProgress } from '@material-ui/core';
+import { Backdrop, CircularProgress, makeStyles } from '@material-ui/core';
 
 import { isManuscriptLoaded } from '../../selectors/manuscript.selectors';
-import './styles.scss';
 import { ManuscriptToolbar } from './manuscript-toolbar';
 import { ManuscriptEditor } from './manuscript-editor';
+import { ManuscriptTOC } from './manuscript-toc';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
```

Comment:
if it's a style of a div we can just use a class from scss. Make styles is good when you have things calculated based on the current theme.